components/movementsensor/errors.go

```diff
@@ -0,0 +1,21 @@
+package movementsensor
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+)
+
+// AddressReadError returns a standard error for when we can not read from an I2C bus.
+func AddressReadError(err error, address byte, bus, board string) error {
+	msg := fmt.Sprintf("can't read from I2C address %d on bus %s of board %s",
+		address, bus, board)
+	return errors.Wrap(err, msg)
+}
+
+// UnexpectedDeviceError returns a standard error we can not find the expected device
```

Comment:
2 typos: "standard error *for when* we *cannot*" (or "error *that* we cannot," or similar)